homework/Vasili_Zasinets/AbstractMath/src/test/java/com/tutrit/java/quickstart/abstractmath/OperationExecutorTest.java

```diff
@@ -0,0 +1,27 @@
+package com.tutrit.java.quickstart.abstractmath;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class OperationExecutorTest {
+
+    MathOperation[] ops = {new AdditionOperation(), new DivisionOperation(), new MultiplicationOperation(),
+            new SubtractionOperation()};
+
+    OperationExecutor executor = new OperationExecutor();
+
+    @Test
+    public void Execute() throws DivisionByZeroException {
+        double a = 8;
+        double b = 4;
+        executor.execute(ops, a, b);
+        Assert.assertNotNull(executor);
+    }
```

Comment:
Let's revise this after mocks, right? I see you don't get what could been tested here ;)

Reply:
ok